doc/How-Tos/DependencyInjection/DependencyInjection-setup.md

```diff
@@ -0,0 +1,88 @@
+# How-To: Use Services with Dependency Injection
+
+Dependency injection (DI) is an important design pattern for building loosely-coupled software that allows for maintainability and testing. This tutorial will walk you through how to register services with a service provider for use throughout your application.
+
+NOTE: This guide assumes you used the Uno.Extensions `dotnet new` template to create the solution. You can verify this by looking for a generated `App.xaml.host.cs` file.
+
+## Step-by-steps
+
+1. Plan the contract for your service
+    * Create a new interface which declares the method(s) your service offers:
+    ```cs
+        public interface IUserProfilePictureService
+        {
+	        Task<byte[]> GetAsync(CancellationToken ct = null);
+        }
+    ```
+
+2. Add and register a service that encapsulates app functionality
+    * Write a new service class which implements the interface you created above, defining its member(s):
+    ```cs
+        public class UserProfilePictureService : IUserProfilePictureService
+        {
+            public async Task<byte[]> GetAsync(CancellationToken ct = null)
+            {
+                . . .
+            }
+        }
+    ```
+    * Register this service implementation with the `IServiceProvider` instance provided by your application's `IHostBuilder`:
```

Comment:
This isn't correct - you register on the IServiceCollection and then resolve on the IServiceProvider instance